### PR TITLE
Update Translation: cli-tooling.adoc.po

### PR DESCRIPTION
**Pull request by amoewaki**

### CODE REVIEW

**Comment by ynojima** [comment]
@amoewaki Thank you for translation as always.
It seems later part is not translated. Could you look into it?
https://pr-preview-624-ja-quarkusio.surge.sh/guides/cli-tooling

**Comment by amoewaki** [comment]
ah maybe the page has been updated

**Comment by amoewaki** [comment]
Updated Translation: please review it.

File: l10n/po/ja_JP/_guides/cli-tooling.adoc.po [hunk 1]
```diff
@@ -511,18 +485,33 @@ msgid ""
...
 msgid "While this document is a useful reference, the client help is the definitive source."
-msgstr "このドキュメントは有用な参考資料ですが、クライアントヘルプが決定的な情報源となります。"
+msgstr "このドキュメントは参照として役立ちますが、最も信頼のおける情報源はクライアントヘルプです。"
 
 #. type: delimited block =
 #: upstream/_guides/cli-tooling.adoc:291
-#, fuzzy
 msgid "If you don't see the output you expect, use `--help` to verify that you are invoking the command with the right arguments."
-msgstr "期待通りの出力が得られない場合は、 `--help` を使って、正しい引数でコマンドを起動しているかどうかを確認してください。"
+msgstr "期待する出力が表示されない場合は、`--help`を使用して、正しい引数を使用してコマンドを呼び出していることを確認してください。"
```
Comment (ynojima) [inline l10n/po/ja_JP/_guides/cli-tooling.adoc.po:514]:
```suggestion
msgstr "期待する出力が表示されない場合は、 `--help` を使用して、正しい引数を使用してコマンドを呼び出していることを確認してください。"
```

**Pull request merged**